Sample/Example-Debug.xcconfig

```diff
@@ -6,4 +6,4 @@
 // override secrets
 
 // uncomment below line to use mocked server
```

Comment:
Is the line `uncomment below line to use mocked server` still needed?

Reply:
Yes. I left it for debugging purposes to easily mock endpoints and test campaigns display in the sample app.
This place is referenced here https://github.com/rakutentech/ios-inappmessaging/blob/21b6df2226d0bf472e4b22da0122cf36a75700fc/Sample/AppDelegate.swift#L17